client/packages/system/src/Patient/Insurance/ListView.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { usePatient } from '../api';
+import { useInsuranceColumns } from './columns';
+import {
+  createQueryParamsStore,
+  createTableStore,
+  DataTable,
+  TableProvider,
+} from '@openmsupply-client/common';
+
+export const InsuranceListView = () => {
+  const columns = useInsuranceColumns();
+  const patientId = usePatient.utils.id();
+  const { data, isLoading } = usePatient.document.insurances(patientId);
+
+  return (
+    <TableProvider
+      createStore={createTableStore}
+      queryParamsStore={createQueryParamsStore({
+        initialSortBy: { key: 'expiryDate' },
```

Comment:
This initial sort by doesn't seem to be working for me?
<img width="1204" alt="image" src="https://github.com/user-attachments/assets/b045e715-4597-4fe4-bf99-6d09293b5377" />